pkg/kubelet/apis/podresources/client.go

```diff
@@ -32,6 +32,11 @@ import (
 // Note: Consumers of the pod resources API should not be importing this package.
 // They should copy paste the function in their project.
 
+const (
+	DefaultTimeout    = 10 * time.Second
+	DefaultMaxMsgSize = 1024 * 1024 * 16 // 16 MiB
```

Comment:
would it be enough for the longest names of pods with the biggest number of Pods?

Reply:
I actually dunno. This is the values copy-pasted from the source code which is constantly passed over. Can't say how this was computed. Likewise, I'm not sure about the formula to estimate the msg size given a pod size. I guess the only option is to watch and react.